packages/Tag/README.md

```diff
@@ -0,0 +1,33 @@
+# Tag
+
+The Tag component informs users of the status of an object or of an action that's been taken.
+
+<Example />
+
+<ReferenceLinks />
+
+## Installation
+
+To install `@igloo-ui/tag` in your project, you will need to run the following command using [npm](https://www.npmjs.com/):
+
+```bash
+npm install @igloo-ui/tag
+```
+
+If you prefer [Yarn](https://classic.yarnpkg.com/en/), use the following command instead:
+
+```bash
+yarn add @igloo-ui/tag
+```
+
+## Usage
+
+Then to use the component in your code just import it!
+
+```jsx
+import Tag from '@igloo-ui/tag';
+
```

Comment:
The import of the icon must be added in the example otherwise copying it will not work will work

```suggestion
import LabelSolid from '@igloo-ui/icons/dist/LabelSolid';

```